woocommerce/woocommerce
# CYS - E2E tests: Add color picker E2E tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds E2E tests for the color picker.

https://github.com/woocommerce/woocommerce/issues/45344



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run ```pnpm run --filter="@woocommerce/plugin-woocommerce" test:e2e-pw plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/color-picker.spec.js```
2. Ensure that everything is green.
3. Change [this value](https://github.com/woocommerce/woocommerce/blob/405dd936f42a49589636b4d6cd1c6ff3b46a9b56/plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/color-palette-variations/constants.ts/#L11-L12) and [this value](https://github.com/woocommerce/woocommerce/blob/405dd936f42a49589636b4d6cd1c6ff3b46a9b56/plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/color-palette-variations/constants.ts/#L1101-L1102).
4. Expect to see that `Picking a color should trigger an update of colors on the site preview` and `Selected color palette should be applied on the frontend` tests fail.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

CYS - E2E tests: Add color picker E2E tests

</details>
